js-practive/js/main.js

```diff
@@ -1 +1,260 @@
-// Place your code here, or you can split it how you what. It's up to you 😎
\ No newline at end of file
+// console.log(event)
+    // 1.очищення всієї сторінки 
+    // 2.умови, що знаходяться в хещі та їх валідація (асинхронний запит який вибудовує сторінку) (вбудувати сторінку )
+let usersTable
+let createButton
+let userForm 
+
+setUsersTable()
```

Comment:
if user opens user form this fn will be executed. It's not good, you have to think about some separation.

Reply:
Maybe better will be to call those functions in `readingHashState` ?